packages/jaeger-ui/src/components/App/TopNav.test.js

```diff
@@ -87,5 +89,11 @@ describe('<TopNav>', () => {
       expect(item.prop('label')).toBe(labelAbout);
       expect(item.prop('items')).toBe(dropdownItems);
     });
+
+    it('adds target=_self to top-level item', () => {
+        const item = wrapper.find(`[href="${githubUrl}"]`);
+        expect(item.length).toBe(1);
+        expect(item.find(`[target="_self"]`).length).toBe(1);
```

Comment:
could we also have a negative test, i.e. if `openInSameTab` not specified, then `target=_blank`

Reply:
You're right, will add this soon.

Reply:
Done